packages/next/client/link.tsx

```diff
@@ -296,6 +306,7 @@ const Link = React.forwardRef<HTMLAnchorElement, LinkPropsReal>(
       locale,
       onClick,
       onMouseEnter,
+      onTouchStart,
```

Comment:
Do we consider using the onPointerEnter to handle all mouse/touch in the future? maybe we can use it to handle touch events first?

Reply:
Are there any quirks to using `onPointerEnter` vs `onMouseEnter`/`onTouchStart`? If not and it handles both definitely sounds ideal

Reply:
I feel the only blocker is browser support. Previously pointer event wasn't supported quite well among all browsers, I remembered years ago safari doesn't. but now it looks not bad, safari 13 has supported it since 2019.

https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent

Reply:
We can investigate using pointer events in a follow-up, seems `onTouchStart` should be a good start for compatibility.